python3_user_features.rst

```diff
@@ -404,3 +404,30 @@ symbols as variable names, e.g.:
 
 Use caution if you're planning to share your code, though, as it's fairly easy
 to produce illegible code this way.
+
+More useful exceptions
```

Comment:
There are, e.g. `FileExistsError` and `PermissionError`. Opening a file was the most obvious case, but not the only one.

I could [link to the Python whatsnew docs](https://docs.python.org/3/whatsnew/3.3.html#pep-3151-reworking-the-os-and-io-exception-hierarchy) for more info, but maybe that's a bit daunting?